crates/cheatcodes/src/evm/fork.rs

```diff
@@ -10,7 +10,6 @@ use foundry_evm_core::fork::CreateFork;
 
 impl Cheatcode for activeForkCall {
     fn apply_full<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
-        let Self {} = self;
```

Comment:
pls keep this for consistency, and it alerts you when the call changes

Reply:
what alerts you?

Reply:
ah the compiler, got it